### PR TITLE
docs: update a jailbroken device section

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

Related to https://github.com/appium/appium/issues/16211

I don't have any jailbroken devices, so we cannot verify the cause.
So, I think it would be nice to address we do not test a jailbroken device and it might have an issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/drivers/ios-xcuitest-real-devices.md [hunk 1]
```diff
@@ -245,8 +245,9 @@ for this).
     1. Settings -> Safari -> Advanced -> **Web Inspector** and **Remote Automation**
         1. Please read [Automating mobile web apps](/writing-running-appium/web/mobile-web) for more details about WebView
 1. Consider generating a provisioning profile with `.xctrunner` identifier if you do not want to generate a wildcard one for manual configuration. The `.xctrunner` config support has been added since Xcode 11. [A reference](https://github.com/appium/appium/issues/13610)
-1. Make sure the device under test is not a jaibroken one
-    - `com.apple.mobile.installation_proxy` service to manage Apps on the device [does not work](https://github.com/appium/appium-desktop/issues/1447)
+1. Make sure the device under test is not a jailbroken one
+    - Possibly it causes an issue such as [`com.apple.mobile.installation_proxy` service error to manage Apps on the device](https://github.com/appium/appium-desktop/issues/1447)
+    - Appium team does not test Appium against jailbroken devives.
```
Comment (mykola-mokhnach) [inline docs/en/drivers/ios-xcuitest-real-devices.md:250]: Appium team does not test xcuitest driver

File: docs/en/drivers/ios-xcuitest-real-devices.md [hunk 2]
```diff
@@ -245,8 +245,9 @@ for this).
     1. Settings -> Safari -> Advanced -> **Web Inspector** and **Remote Automation**
         1. Please read [Automating mobile web apps](/writing-running-appium/web/mobile-web) for more details about WebView
 1. Consider generating a provisioning profile with `.xctrunner` identifier if you do not want to generate a wildcard one for manual configuration. The `.xctrunner` config support has been added since Xcode 11. [A reference](https://github.com/appium/appium/issues/13610)
-1. Make sure the device under test is not a jaibroken one
-    - `com.apple.mobile.installation_proxy` service to manage Apps on the device [does not work](https://github.com/appium/appium-desktop/issues/1447)
+1. Make sure the device under test is not a jailbroken one
+    - Possibly it causes an issue such as [`com.apple.mobile.installation_proxy` service error to manage Apps on the device](https://github.com/appium/appium-desktop/issues/1447)
+    - Appium team does not test xcuitest driver against jailbroken devives.
```
Comment (mykola-mokhnach) [inline docs/en/drivers/ios-xcuitest-real-devices.md:250]: I'd say it makes sense to change the top sentence though

Appium team does not test xcuitest driver against jailbroken devices, so we cannot guarantee it is going to work as expected there. Use it at your own risk.

Comment (KazuCocoa) [reply docs/en/drivers/ios-xcuitest-real-devices.md:250]: good idea

Comment (KazuCocoa) [reply docs/en/drivers/ios-xcuitest-real-devices.md]: Changed c81e233

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**